interview-questions.md

```diff
@@ -8,52 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is a hash in Ruby?
 
   Your answer:
+  In Ruby a hash is a collection of key:value pairs. We worked with hashes during our hashes coding challenges.
 
   Researched answer:
-
-
+  In Ruby a hash is a container of key: value pairs. To access the values in a hash we use CRUD (create, read, update, delete) actions that can manipulate the data in different ways by referencing the keys in a hash. Hashes are similar to arrays and ranges in that they are enumerable, which means methods such as .each and .map can be used to iterate over a hash.
 
 2. What is a gem?
 
   Your answer:
+  In Ruby a gem is a package manager. We used the Active Record gem when doing our coding challenges in that section.
 
   Researched answer:
-
+  A gem is a open source library in Ruby that adds functionality to a program. A gem allows programmers to use the Ruby code that is in the library rather than having to insert the code into their program. 
 
 
 3. What is Ruby on Rails?
 
   Your answer:
+  Ruby on Rails is a collection of gems built in the Ruby language and it is a fullstack web application framework.  
```

Comment:
Could you share a brief explanation of a project that you created using Ruby on Rails?